test/integration/src/main/java/io/pravega/test/integration/selftest/StreamProducerDataSource.java

```diff
@@ -258,5 +261,13 @@ private void postStreamDeletion(String name) {
         this.state.recordDeletedStream(name);
     }
 
+    private CompletableFuture<Void> deleteStreamThrowing(String name, Duration timeout) {
+        try {
+            return this.store.deleteStream(name, this.config.getTimeout());
+        } catch (TxnFailedException ex) {
+            throw new CompletionException(ex);
```

Comment:
return a failed future  here and in other points

Reply:
Yes Enrico